src/patterns.js

```diff
@@ -99,6 +99,7 @@ define([
     // Since we are in a non-AMD env, register a few useful utilites
     var window = require("window");
     window.jQuery = $;
+    window.patterns = registry;
```

Comment:
This is only to easily instantiate the modal pattern via ``patterns.patterns.modal``. We have the same in https://github.com/quaive/ploneintranet.prototype/blob/01e4cb5c3ac2690816d5e1716ffa32c5dafef7bf/bundle-config.js#L88